Source/Core/Common/GL/GLInterface/EGLDRM.cpp

```diff
@@ -0,0 +1,1287 @@
+// Copyright 2020 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
```

Comment:
Top level comment for the file just to state that a lot of this code currently doesn't match our naming conventions and coding styles and should conform to that.

A lot of it is very C89 (variables at top of scopes, rather than where they're used, etc) and should be migrated to idiomatic C++ before being merged.